src/collective/cover/tests/test_purge_data.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+Library  ${CURDIR}/TestInternalServerError.py
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_location}  'collective.cover.basic'
+${title_field_id}  collective-cover-basic-title
+${title_sample}  Some text for title
+${edit_link_selector}  a.edit-tile-link
+
+*** Test cases ***
+
+Test Purge Data
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+    Create Cover  Title  Description
+
+    # add tile to the layout
+    Open Layout Tab
+    Add Tile  ${basic_tile_location}
+    Save Cover Layout
+
+    # go to compose view and edit the tile
+    Compose Cover
+
+    # keep edit url and compose url for later use
+    ${compose_url} =  Execute Javascript
+    ...  return (function() {
+    ...    return location.href;
+    ...  })();
+    ${edit_url} =  Execute Javascript
+    ...  return (function() {
+    ...    var $link = jQuery('${edit_link_selector}');
+    ...    return $link.attr('href');
+    ...  })();
+
+    Click Link  css=${edit_link_selector}
+    Wait Until Page Contains  Edit Basic Tile
+    Input Text  id=${title_field_id}  ${title_sample}
+    Click Button  Save
+
+    # check for successful AJAX refresh
+    Wait Until Page Contains  ${title_sample}
+
+    # confirm that data is still there
+    Go to  ${edit_url}
+    ${value} =  Get value  ${title_field_id}
+    Should be equal  ${title_sample}  ${value}
+
+    Go to  ${compose_url}
+    Open Layout Tab
+    Delete Tile
+    Save Cover Layout
+
+    # confirm that data is gone
+    Go to  ${edit_url}
+    ${value} =  Get value  ${title_field_id}
+    Should Not be equal  ${title_sample}  ${value}
```

Comment:
In fact, it needs to be empty. They are different things.

Reply:
That's why I use `Should Not be equal`